### PR TITLE
Add composed task indicator to TaskDefinitionResource

### LINKED ISSUE #1313: Add composed task indicator
Depends on #1302

### PR DESCRIPTION
**Pull request by trisberg**
- store original dsl

- calculate executable graph on launch

- calculate composed indicator when creating TaskDefinitionResource

Resolves #1313

### CODE REVIEW

**Comment by trisberg** [comment]
For UI changes see https://github.com/spring-cloud/spring-cloud-dataflow-ui/pull/196

**Comment by cppwfs** [comment]
LGTM

File: spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/services/impl/DefaultTaskServiceTests.java [hunk 1]
```diff
@@ -192,9 +192,9 @@ public void createSequenceComposedTask() {
 	@Test
 	@DirtiesContext
 	public void createSplitComposedTask() {
-		taskService.saveTaskDefinition("splitTask", "<AAA || BBB>");
-		verifyTaskExistsInRepo("splitTask",
-				"<splitTask-AAA || splitTask-BBB>");
+		String dsl = "<AAA || BBB>";
+		taskService.saveTaskDefinition("splitTask", dsl);
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/services/impl/DefaultTaskServiceTests.java]: Would be good if we also have some tests for `taskService.isComposedDefinition(dsl)` as well. Will add when merging.

**Comment by ilayaperumalg** [comment]
LGTM as well, merging with some test updates.

**Comment by ilayaperumalg** [comment]
Merged as 384ce199527bf3c56f6fcb0f80c09dde3fc2e121 along with 99f05207e1d3f7c3576d96dcd229bc7a25bb5ac7

**Pull request merged**